### PR TITLE
2 taps to open it.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,11 @@ This application is inspired by the apiMate.
 ## Usage
 
 ![Usage](docs/usage.gif)
+
+or
+
+```Text
+APIMateR apimater://{$host},{$bbs},{$key}{$host[match:\.((2ch|5ch)\.net|bbspink\.com)$]}
```
Comment (nonnonstop) [inline README.md]:
```suggestion
APIMateR apimater:https://{$host}/test/read.cgi/{$bbs}/{$key}/{$host[match:\.([25]ch\.net|bbspink\.com)$]}
```

File: app/src/main/java/com/nonnonstop/apimate/LoadActivity.kt [hunk 2]
```diff
@@ -24,7 +24,12 @@ class LoadActivity : AppCompatActivity() {
     }
 
     private fun dispatchIndent(intent: Intent?) {
-        val htmlUrl = intent?.dataString ?: return
+        val htmlUrl = if (intent?.scheme == "apimater") {
+            val (host, bbs, key) = (intent.data?.host ?: return).split(",")
+            "https://$host/test/read.cgi/$bbs/$key/"
+        } else {
+            intent?.dataString ?: return
+        }
```
Comment (nonnonstop) [inline app/src/main/java/com/nonnonstop/apimate/LoadActivity.kt]:
```suggestion
        val htmlUrl = intent?.data?.normalizeScheme()?.run {
            if (scheme == "apimater")
                encodedSchemeSpecificPart
            else
                toString()
        } ?: return
```

**Review by nonnonstop — CHANGES_REQUESTED** [review]
can be more generic

**Comment by ghost** [comment]
I'm sorry. Fixed.

**Review by nonnonstop — APPROVED** [review]
approved. thank you!

**Pull request merged**